src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -697,6 +697,35 @@ namespace webnn_native { namespace ie {
         std::vector<int32_t> outputPadding(options->outputPadding,
                                            options->outputPadding + options->outputPaddingCount);
         DAWN_ASSERT(outputPadding.size() == 2);
+        // (workaround) pads values are generated via the autoPad option:
```

Comment:
Maybe we should disable the test case or not support debug version CI, but not add the workaround if it's OpenVINO's bug.